.github/workflows/checks-quick.yml

```diff
@@ -172,6 +171,27 @@ jobs:
         env:
           ASSERT_REGEX: "FAIL-CI"
           GIT_DEPTH: 1
+  check-readme:
```

Comment:
The job fails on the CI: https://github.com/paritytech/polkadot-sdk/actions/runs/11776348092/job/32798664416?pr=6359. Seems like we need to install protobuf. We can add something like:
```
      - name: Rust compilation prerequisites
        run: |
          sudo apt update
          sudo apt install -y \
            protobuf-compiler
```